wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioRecordKeyboardViewController.swift

```diff
@@ -191,27 +191,27 @@ final class AudioRecordKeyboardViewController: UIViewController, AudioRecordBase
         self.recordButton.setIcon(.recordDot, size: .tiny, for: [])
         self.recordButton.setIconColor(.white, for: [])
         self.recordButton.addTarget(self, action: #selector(recordButtonPressed), for: .touchUpInside)
-        self.recordButton.setBackgroundImageColor(SemanticColors.LegacyColors.vividRed, for: .normal)
+        self.recordButton.setBackgroundImageColor(SemanticColors.Icon.foregroundDefaultRed, for: .normal)
         self.recordButton.layer.masksToBounds = true
 
         self.stopRecordButton.setIcon(.stopRecording, size: .tiny, for: [])
         self.stopRecordButton.setIconColor(.white, for: [])
         self.stopRecordButton.addTarget(self, action: #selector(stopRecordButtonPressed), for: .touchUpInside)
-        self.stopRecordButton.setBackgroundImageColor(SemanticColors.LegacyColors.vividRed, for: .normal)
+        self.stopRecordButton.setBackgroundImageColor(SemanticColors.Icon.foregroundDefaultRed, for: .normal)
         self.stopRecordButton.layer.masksToBounds = true
 
         self.confirmButton.setIcon(.checkmark, size: .tiny, for: [])
         self.confirmButton.setIconColor(.white, for: [])
         self.confirmButton.addTarget(self, action: #selector(confirmButtonPressed), for: .touchUpInside)
-        self.confirmButton.setBackgroundImageColor(SemanticColors.LegacyColors.strongLimeGreen, for: .normal)
+        self.confirmButton.setBackgroundImageColor(SemanticColors.Icon.backgroundJoinCall, for: .normal)
```

Comment:
Also this seems like a misuse of a semantic color.

Reply:
Done!